<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportF16</class>
 <widget class="QDialog" name="ReportF16">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="0" colspan="3">
    <widget class="CRBListBox" name="lstOrgStructure"/>
   </item>
   <item row="8" column="1" colspan="2">
    <widget class="CRBComboBox" name="cmbProfileBed"/>
   </item>
   <item row="10" column="0" colspan="3">
    <widget class="CRBListBox" name="lstProfileBed"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Тип финансирования</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="COrgStructureComboBox" name="cmbOrgStructure"/>
   </item>
   <item row="8" column="0">
    <widget class="QCheckBox" name="chkProfileBed">
     <property name="text">
      <string>Профиль</string>
     </property>
    </widget>
   </item>
   <item row="12" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QComboBox" name="cmbOrder">
     <item>
      <property name="text">
       <string>не задано</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>планово</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>экстренно</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QComboBox" name="cmbTypeHosp">
     <item>
      <property name="text">
       <string>не задано</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>круглосуточный стационар</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>дневной стационар</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>амбулаторно</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="CDateEdit" name="edtBegDate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblBegDate">
     <property name="text">
      <string>Дата &amp;начала периода</string>
     </property>
     <property name="buddy">
      <cstring>edtBegDate</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblEndDate">
     <property name="text">
      <string>Дата &amp;окончания периода</string>
     </property>
     <property name="buddy">
      <cstring>edtEndDate</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblOrder">
     <property name="text">
      <string>Порядок</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="chkOrgStructure">
     <property name="text">
      <string>Подразделение</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="CRBComboBox" name="cmbFinance"/>
   </item>
   <item row="1" column="1">
    <widget class="CDateEdit" name="edtEndDate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblTypeHosp">
     <property name="text">
      <string>Тип госпитализации</string>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit.h</header>
  </customwidget>
  <customwidget>
   <class>COrgStructureComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgStructComboBoxes.h</header>
  </customwidget>
  <customwidget>
   <class>CRBListBox</class>
   <extends>QListView</extends>
   <header>library/RBListBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ReportF16</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ReportF16</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
